packages/babel-traverse/test/modification.js

```diff
@@ -78,6 +98,26 @@ describe("modification", function () {
         },
       });
     });
+
+    it("should set the correct path.context", function () {
+      expect.assertions(2);
```

Comment:
Neat!

Reply:
We should probably add this to all the tests where we `expect` inside a visitor, just in case the visitor doesn't get called!